### PR TITLE
Refine describe index that doesn't wait compacted segment indexed

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #25891

### CODE REVIEW

**Comment by xiaocai2333** [comment]
/close
optimize on master

File: internal/indexcoord/index_coord.go [hunk 1]
```diff
@@ -605,7 +605,7 @@ func (i *IndexCoord) getIndexIDAndCreateTime(collectionID int64, indexName strin
 // completeIndexInfo get the index row count and index task state
 // if realTime, calculate current statistics
 // if not realTime, which means get info of the prior `CreateIndex` action, skip segments created after index's create time
-func (i *IndexCoord) completeIndexInfo(indexInfo *indexpb.IndexInfo, index *model.Index, segIDs []UniqueID) error {
+func (i *IndexCoord) completeIndexInfo(indexInfo *indexpb.IndexInfo, index *model.Index, segIDs []UniqueID, realTime bool) error {
```
Comment (czs007) [inline internal/indexcoord/index_coord.go:608]: better to rename as filterByTime bool

Comment (czs007) [reply internal/indexcoord/index_coord.go:608]: you can change this in next pr

**Pull request merged**